TRANSFORMERS/TYPE_CASTING/DF_2_NP.py

```diff
@@ -0,0 +1,23 @@
+import traceback
+from numpy import asmatrix
+from pandas import to_numpy
+from flojoy import flojoy, DataContainer
+
+
+@flojoy
+def DF_2_NP(
+    dc_inputs: list[DataContainer(type="dataframe")], params: dict
+) -> DataContainer:
+    """
+    Node to convert dataframe type data into matrix type data.
+    It takes one dataframe type data and converts it to matrix type data.
+    """
+    try:
+        pdData = dc_inputs[0].m
```

Comment:
Instead of using try/except you should probably check that the input type is `'dataframe'`, and raise an exception if it isn't. This lets you specify a more user friendly error message rather than just displaying the traceback.